repondeur/tests/test_lectures_list.py

```diff
@@ -13,23 +8,11 @@ def test_get_list_empty(app):
     assert "Aucune lecture pour l'instant." in resp.text
 
 
-@pytest.yield_fixture
-def dummy_lecture(app):
-    data_dir = app.app.registry.settings["zam.data_dir"]
-    dir_names = ["senat-2017-2018-63", "ignore_this"]
-    paths = [os.path.join(data_dir, dir_name) for dir_name in dir_names]
-    for path in paths:
-        os.makedirs(path)
-    yield
-    for path in paths:
-        os.rmdir(path)
-
-
 def test_get_list_not_empty(app, dummy_lecture):
 
     resp = app.get("/lectures/")
 
     assert resp.status_code == 200
     assert resp.content_type == "text/html"
 
-    assert "Sénat, session 2017-2018, texte nº 63" in resp.text
+    assert "Assemblée nationale, session 15, texte nº 0269" in resp.text
```

Comment:
Makes me wonder if we want to left strip zeros on text numbers.

Reply:
I think those pseudo-titles will go away soon, and we'll just use the real title from the *projet de loi* instead.

Reply:
I did some clean up anyway, and made sure all the texte and amendement numbers are integers now → 88f3bda044f6340a2262764e7de716f896ee374f